coinbase_tests.py

```diff
@@ -151,9 +150,51 @@ def tearDown(self):
         self.page.driver.quit()
 
 
+class HomePageSetupTemplate(unittest.TestCase):
 
+    def setUp(self):
+        self.driver = webdriver.Chrome(root_path + chromedriver_path)
+        self.page = HomePage(self.driver)
+    
+    def tearDown(self):
 
+        self.page.driver.quit()
 
 
+class HomePageProducts(HomePageSetupTemplate):
+    """
+    Confirm the list of the company's featured products are present on
+    homepage. Ensure that clicking on each product link will navigate to the
+    dialogue box 'create a new account'.
+
+    acceptance cirteria
+    --------------------
+    -All of the provided featured products are listed.
+    -When clicking on each of the product's link it opens the "create new account
+     dialogue box".   
+    """
+
+    def testProvidedProducts(self):
+        #below is a mock data set representing a source of truth 
+        provided_featured_products = {'Bitcoin\nBTC', 'Ethereum\nETH', 
+                                      'Bitcoin Cash\nBCH', 'Litecoin\nLTC'} 
+
+        #featured products retrieved from coinbase
+        featured_products = set() 
+        for element in self.page.featured_products:
+            featured_products.add(element.text)
+        
+        self.assertEqual(featured_products, provided_featured_products)
+
+    def testProductsLink(self):
+        page = self.page
+        
+        for product in page.products_links:
+            product.click()
+            self.assertTrue(page.account_dial_box.exists(),
```

Comment:
need to use full words. "account_dialogue_box"

Someone could guess "account_dialogue_box" but they're unlikely to guess it if there are random abbreviations.

(note: it's not a dialogue box, it's just a dialogue, so "account_dialogue" would do... but _box is fine)